packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/isShippingRestricted.js

```diff
@@ -0,0 +1,53 @@
+import operators from "@reactioncommerce/api-utils/operators.js";
```

Comment:
Needs tests

Reply:
Tests are present in `filterShippingMethods.test.js`